src/auxiliary/Debug.hh

```diff
@@ -25,46 +35,42 @@ public:
     static void off() { Debug::debug_ = false; }
 
     template <typename scalar_t>
-    static void diffLapackMatrices(int64_t m, int64_t n,
-                                   scalar_t const* A, int64_t lda,
-                                   scalar_t const* B, int64_t ldb,
-                                   int64_t mb, int64_t nb);
+    static void diffLapackMatrices(
+        int64_t m, int64_t n,
+        scalar_t const* A, int64_t lda,
+        scalar_t const* B, int64_t ldb,
+        int64_t mb, int64_t nb );
+
     //-------------
     // BaseMatrix class
     template <typename scalar_t>
-    static void checkTilesLives(BaseMatrix<scalar_t> const& A);
-
-    template <typename scalar_t>
-    static bool checkTilesLayout(BaseMatrix<scalar_t> const& A);
-
-    template <typename scalar_t>
-    static void printTilesLives(BaseMatrix<scalar_t> const& A);
+    static void checkTilesLives( BaseMatrix<scalar_t> const& A );
 
     template <typename scalar_t>
-    static void printTilesMaps(BaseMatrix<scalar_t> const& A);
+    static bool checkTilesLayout( BaseMatrix<scalar_t> const& A );
 
     template <typename scalar_t>
-    static void printTilesMOSI(BaseMatrix<scalar_t> const& A, const char* name,
-                               const char* func, const char* file, int line);
+    static void printTiles_(
+        BaseMatrix<scalar_t> const& A, const char* name, int fields,
+        const char* func, const char* file, int line );
 
-    #define PRINTTILESMOSI(A) \
-            printTilesMOSI(A, #A, __func__, __FILE__, __LINE__);
+    #define printTiles( A, fields ) \
```

Comment:
I am not sure that is a good idea to replace all capital letters with lower case letters.

Reply:
I avoid ALLCAPS because it is incredibly hard to read. Here `printTiles` gets used as a global function, which should probably actually be `snake_case`, but everything else in Debug is `camelCase`. Many macros are lowercase: slate_assert, slate_mpi_call, scalapack_p*, and in the C library, assert, va_start, va_arg, va_end, setjmp.

Reply:
Actually it appears like a class function, `Debug::printTiles`, which would still be `camelCase` (though I wish they were `snake_case` too).